apps/penxle.com/src/lib/components/Tag.svelte

```diff
@@ -11,6 +13,7 @@
     size === 'sm' && 'px-3 h-6.5 text-3.25',
     size === 'lg' && 'py-1 px-4 h-8 text-3.75',
     state === 'select' && 'border-gray-90',
+    _class,
```

Comment:
```suggestion
    'flex items-center,
    _class
```

P3: 태그 컴포넌트가 인라인 정렬이라면 위같이 내부에 넣는게 나아보이네요.